packages/charts/src/charts/line/components/tooltip/tooltip.component.tsx

```diff
@@ -20,9 +24,10 @@ const Tooltip: FC<Props> = ({ data, selectors, formatValue }) => {
   return (
     <BulletList
       items={selectors.map(({ color, selector }) => ({
-        data: formatValue
-          ? formatValue(getFromPath(data, selector))
-          : getFromPath(data, selector),
+        data: valueFormatter(
```

Comment:
I believe @mczapkowicz will need to address this use case on his branch. As tooltips re-work is in progress for this chart.